### PR TITLE
Prepare release of GAE-storage v0.7.0

### PR DESCRIPTION
**Pull request by dmdashenkov**
- Update README.md.
 - Fix emulator startup script crossplatformness.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: README.md [hunk 1]
```diff
@@ -6,16 +6,34 @@
...
- - to run tests you should either select `spine-dev` Google Cloud Console project or create one if it's not accessible for you 
- - if the Google project has different name, change it in `LocalDatastoreStorageFactory`
- 
-The launched emulator will run in `localhost:8080` and will not have any persistence.
-The datastore is cleaned up after each test.
+ - to run tests you should select `spine-dev` Google Cloud Console project
+ - skip Google App Engine setup if not required
+
+
+###### Unix-like
```
Comment (armiol) [inline README.md]: `Unix-based`

File: README.md [hunk 2]
```diff
@@ -6,16 +6,34 @@
...
+To start a localhost emulator and run tests run `./gradlew check`.
```
Comment (armiol) [inline README.md]: `To start a local emulator and run test against it, run ....`

File: README.md [hunk 3]
```diff
@@ -6,16 +6,34 @@
...
+To start an emulator without running tests `./gradlew startDatastore`.
+To stop datastore use standard system means (e.g. `kill -9 $(lsof -i:8080)`).
```
Comment (armiol) [inline README.md]: `To stop the Datastore emulator, just terminate the emulator process. (e.g. ...).

File: README.md [hunk 4]
```diff
@@ -6,16 +6,34 @@
...
+
+###### Windows
+
+To start a localhost emulator go to dir `script` and run `start-datastore.bat` as an __administrator__.
```
Comment (armiol) [inline README.md]: Same as above.

File: README.md [hunk 5]
```diff
@@ -6,16 +6,34 @@
...
+The first launch may download and initialize the emulator itself. If so, rerun the script after the install is complete.
+To run tests execute `gradlew check`
+
+###### General
+
+The launched emulator will run at `localhost:8080` and will not have any persistence.
 To change this configs see `./script/start-datastore.*` scripts.
```
Comment (armiol) [inline README.md]: `this configs` -> `the configuration`

File: build.gradle [hunk 6]
```diff
@@ -165,7 +165,7 @@ startDatastore << {
     }
 
     // Run datastore emulator
-    "$pathPrefix/script/start-datastore$scriptExtension".execute()
+    "${pathPrefix}${File.separatorChar}script${File.separatorChar}start-datastore$scriptExtension".execute()
```
Comment (armiol) [inline build.gradle]: Please ensure that works; ask @dmytro-kashcheiev for an assistance if needed.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov please see my comments.

**Comment by dmdashenkov** [comment]
@armiol PTAL at the latest changes

File: README.md [hunk 7]
```diff
@@ -6,16 +6,28 @@
...
-To stop datastore use standard system means (e.g. `kill -9 \$(lsof -i:8080)`).
-To run the task successfully, you must have `gcloud` tool properly
-installed and configured: 
+
+##### Testing
+
+###### Preconditions
+
+To run the task successfully, you must have `gcloud` tool properly installed and configured: 
  - install gcloud of the last version
```
Comment (armiol) [inline README.md]: Please add either `;` or `,` at the end of the items.
The last item in the list should end with a dot.

File: build.gradle [hunk 8]
```diff
@@ -154,18 +154,14 @@ task startDatastore {
...
-        scriptExtension = ".bat"
-        pathPrefix = ""
+        final command = "${project.rootDir.absolutePath}\\script\\start-datastore.bat"
+        println command
+        command.execute()
     } else {
-        scriptExtension = ".sh"
-        pathPrefix = "."
+        // Run datastore emulator
+        ".${File.separatorChar}script${File.separatorChar}start-datastore.sh".execute()
```
Comment (armiol) [inline build.gradle]: Why not have the same code for Windows and Unix?
The only difference is the extension of the script. That's what you set depending on `IS_WINDOWS`.

**Review by armiol — CHANGES_REQUESTED** [review]
@dmdashenkov please see my comments.

**Comment by dmdashenkov** [comment]
@armiol PTAL again

**Review by armiol — APPROVED** [review]
@dmdashenkov LGTM.

**Pull request merged**